lib/components.js

```diff
@@ -363,7 +363,9 @@ class DatePicker extends Component {
       'minimumDate',
       'minuteInterval',
       'mode',
-      'timeZoneOffsetInMinutes'
+      'timeZoneOffsetInMinutes',
+      'background',
```

Comment:
> I'm always in favor of removing nesting

:+1:

> using the mode option of DatePickerIOS to set the time or date picker in Android (both do conceptually the same)

:+1:

> and also put the format option in the same level

Another important benefit of implementing a generic `config` option (which could be added to `Component`'s locals instead of `DatePicker`) is allowing the users of `tcomb-form-native` to pass custom options to their custom templates (as you are doing here with `background` and `format`).
Currently the `tcomb-form-native` templating system seems to miss this feature. If I write my own template for the android date picker it's possible (even probable) that `background` and `format` are not enough, maybe I don't need them, maybe I need **more** options.